crates/nu-utils/src/sample_config/default_config.nu

```diff
@@ -152,6 +152,7 @@ $env.config = {
     ls: {
         use_ls_colors: true # use the LS_COLORS environment variable to colorize output
         clickable_links: true # enable or disable clickable links. Your terminal has to support links.
+        sort_by: [] # Adds sorting on the output of ls, Example: sort_by: ['name reverse natural', 'type'] would be similar to `ls | | sort-by name -rn | sort-by type`
```

Comment:
I'm also wondering if `sort-by name -rn | sort-by type` accurate. I thought sort-by could now sort multiple columns at the same time versus sort then sort again. There was a PR recently that overhauled the sorting to make it better.

Reply:
You're right actually, I missed this. The implementation of this PR will do the equivalent of `sort-by -rn name type`, which will sort naturally for both name and type and then will reverse. You can't specify per comparator natural or reverse, you would have to run multiple `sort_by`s of one comparator each.